ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -425,26 +417,64 @@ impl<'a> Assemble<'a> {
             #[cfg(debug_assertions)]
             infos.sort_by(|a, b| a.0.cmp(&b.0));
 
+            let deopt_label = self.asm.new_dynamic_label();
             let guardcheck_label = self.asm.new_dynamic_label();
             for (deoptid, fail_label) in infos {
                 self.comment(
                     self.asm.offset(),
-                    format!("Deopt ID for guard {:?}", deoptid),
+                    format!("Deopt ID and patch point for guard {:?}", deoptid),
                 );
                 self.ra
                     .get_ready_for_deopt(&mut self.asm, &self.deoptinfo[&deoptid].0);
                 // FIXME: Why are `deoptid`s 64 bit? We're not going to have that many guards!
+
+                // Align this location to the cache-line, which since a cache-line is a multiple of
+                // 8 bytes, will also word-align it too. This allows us to safely patch this place
+                // later in a single atomic operation.
+                let clsize =
+                    cache_size::cache_line_size(1, cache_size::CacheType::Instruction).unwrap();
+                // Unwrap can only fail if there's an overflow.
```

Comment:
I don't think there's an `unwrap` that this refers to?

Oh, hang on, is it referring to the line *above`? But I don't see overflow happening anywhere?

Reply:
Think that's an old comment.